playwright-e2e/pages/page-base.ts

```diff
@@ -105,4 +112,18 @@ export default abstract class PageBase implements PageInterface {
   async logOut(): Promise<void> {
     await Promise.all([this.page.waitForNavigation({ waitUntil: 'load' }), this.getLogOutButton().click()]);
   }
+
+  /**
+   * <br> Question: Date of Birth
+   * <br> Type: Input
+   * @param month
+   * @param date
+   * @param year
+   */
+  async fillInDateOfBirth(month: number | string, date: number | string, year: number | string): Promise<void> {
+    const dob = new Question(this.page, { prompt: 'Date of Birth' });
+    await dob.date().locator('input[data-placeholder="MM"]').fill(month.toString());
+    await dob.date().locator('input[data-placeholder="DD"]').fill(date.toString());
+    await dob.date().locator('input[data-placeholder="YYYY"]').fill(year.toString());
+  }
```

Comment:
Copied from `consent-form-page.ts`. Prefixed name with a verb to indicate an action.